src/actions/format_actions.ts

```diff
@@ -256,6 +256,46 @@ export const formatUnderline: ActionSpec = {
   isActive: (env) => !!env.model.getters.getCurrentStyle().underline,
 };
 
+export const formatRotation: ActionSpec = {
+  name: _t("Rotation"),
+  icon: (env) => getRotationIcon(env),
+};
+
+export const formatNoRotation: ActionSpec = {
+  name: _t("No Rotation"),
+  execute: (env) => setStyle(env, { rotation: 0 }),
+  icon: "o-spreadsheet-Icon.ROTATION-0",
+  isActive: (env) => env.model.getters.getCurrentStyle().rotation === 0,
+};
+
+export const formatRotation45: ActionSpec = {
+  name: _t("45° Rotation"),
+  execute: (env) => setStyle(env, { rotation: Math.PI / 4 }),
+  icon: "o-spreadsheet-Icon.ROTATION-45",
+  isActive: (env) => env.model.getters.getCurrentStyle().rotation === Math.PI / 4,
+};
+
+export const formatRotation90: ActionSpec = {
+  name: _t("90° rotation"),
```

Comment:
```suggestion
  name: _t("45° rotation"),
  execute: (env) => setStyle(env, { rotation: Math.PI / 4 }),
  icon: "o-spreadsheet-Icon.ROTATION-45",
  isActive: (env) => env.model.getters.getCurrentStyle().rotation === Math.PI / 4,
};

export const formatRotation90: ActionSpec = {
  name: _t("90° rotation"),
```
capital letters consistency.
I think we generally only capitalize the first letter of the first word